src/Tomcat/TomcatController.ts

```diff
@@ -103,7 +103,7 @@ export class TomcatController {
         const newName: string = await vscode.window.showInputBox({
             prompt: 'input a new server name',
             validateInput: (name: string): string => {
-                if (!name.match(/^[\w.-]+$/)) {
+                if (!name.match(/^[\w.-]+$/) || this._tomcatModel.getTomcatServer(name)) {
```

Comment:
we can seperate the validation into two if block. This will make the error message more specific